core/test/unit/middleware/spam-prevention_spec.js

```diff
@@ -19,6 +20,7 @@ describe('Middleware: spamPrevention', function () {
         spyNext = sinon.spy(function (param) {
             error = param;
         });
+        middleware.spamPrevention = rewire('../../../server/middleware/spam-prevention').__get__('spamPrevention');
```

Comment:
suggestion:
what you actually want to achieve is restting `loginSecurity` to `[]`.

```
var spamPrevention  = rewire('../../../server/middleware/spam-prevention');


afterEach
    spamPrevention.__set__('loginSecurity', []);
```